test/e2e/bundle_e2e_test.go

```diff
@@ -34,19 +41,49 @@ var _ = Describe("Installing bundles with new object types", func() {
 	})
 
 	When("a bundle with a pdb, priorityclass, and VPA object is installed", func() {
-		By("including the VPA CRD in the CSV")
 		const (
 			packageName = "busybox"
 			channelName = "alpha"
 			subName     = "test-subscription"
 		)
+		var u unstructured.Unstructured
 
 		BeforeEach(func() {
+			By("first installing the VPA CRD on cluster")
 			const (
 				sourceName = "test-catalog"
 				imageName  = "quay.io/olmtest/single-bundle-index:pdb"
 			)
 
+			// create VPA CRD on cluster
+			y, err := ioutil.ReadFile("data/vpa/crd.yaml")
```

Comment:
this can't be found in the upstream tests. I suspect we'll need to pull it into the test binary with go `bindata`.